packages/types/lib/connection/api/get.ts

```diff
@@ -68,19 +62,33 @@ export type GetConnection = Endpoint<{
     Querystring: {
         env: string;
         provider_config_key: string;
-        force_refresh?: 'true' | 'false' | undefined;
     };
     Path: '/api/v1/connections/:connectionId';
-    Error:
-        | ApiError<'unknown_connection'>
-        | ApiError<'missing_provider_config'>
-        | ApiError<'unknown_provider'>
-        | ApiError<'missing_connection'>
-        | ApiError<'unknown_provider_config'>;
+    Error: ApiError<'unknown_provider_config'>;
     Success: {
-        provider: string | null;
-        connection: Connection;
-        errorLog: ActiveLog | null;
+        data: {
+            provider: string | null;
+            connection: ApiConnectionFull;
+            endUser: ApiEndUser | null;
+            errorLog: ActiveLog | null;
+        };
+    };
+}>;
+export type PostConnectionRefresh = Endpoint<{
+    Method: 'POST';
+    Params: {
+        connectionId: string;
+    };
+    Querystring: {
+        env: string;
+        provider_config_key: string;
+    };
+    Path: '/api/v1/connections/:connectionId/refresh';
+    Error: ApiError<'unknown_provider_config'> | ApiError<'failed_to_refresh', any, ActiveLog | null>;
+    Success: {
+        data: {
+            success: boolean;
+        };
```

Comment:
I am not super convinced by the type names between `ApiConnectionSimple`, `ApiConnectionFull`, `ApiPublicConnection`, there are not super descriptive but I cannot think of something better.

Reply:
yep I agree, usually I go with some kind of variant like this but never found a satisfying name. 
Alternatively, it could be named ApiGetConnectionsItem and ApiGetConnectionItem or something like that ^^' 

it's either too verbose or too vague